### PR TITLE
Fixes for golden path Settlements (in scope of migration to settlements V2).

### PR DESCRIPTION
**Pull request by eugenepolschikov**
1) implemented settlement model creation
2) added 'settlementModel' as parameter for payload
3) updated postman local environment with HOST_TRANSACTION_REQUESTS_SERVICE http://transaction-request-service.local

### CODE REVIEW

File: Golden_Path.postman_collection.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"info": {
-		"_postman_id": "1e21d368-aee7-4fbd-894a-1222c8ab37d7",
-		"name": "Golden_Path",
+		"_postman_id": "09a47d32-319f-4ecc-8cdc-61a1e606585b",
+		"name": "Golden_Path copy",
```
Comment (ndonnan) [inline Golden_Path.postman_collection.json]: Please remove "copy" from the collection name

File: OSS-New-Deployment-FSP-Setup.postman_collection.json [hunk 2]
```diff
@@ -416,6 +416,77 @@
 			],
 			"protocolProfileBehavior": {}
 		},
+		{
+			"name": "SettlementModels",
```
Comment (ndonnan) [inline OSS-New-Deployment-FSP-Setup.postman_collection.json]: I think we should call this folder "Add Settlement Models" for consistency

File: Golden_Path.postman_collection.json (hunk 1 above)
Comment (eugenepolschikov) [reply Golden_Path.postman_collection.json:4]: @ndonnan . done, adjusted in a new cleaned up PR

File: OSS-New-Deployment-FSP-Setup.postman_collection.json (hunk 2 above)
Comment (eugenepolschikov) [reply OSS-New-Deployment-FSP-Setup.postman_collection.json:420]: done, adjusted naming.

Comment (eugenepolschikov) [reply OSS-New-Deployment-FSP-Setup.postman_collection.json:420]: adjusted in a follow-up PR: https://github.com/mojaloop/postman/pull/100

File: Golden_Path.postman_collection.json (hunk 1 above)
Comment (eugenepolschikov) [reply Golden_Path.postman_collection.json:4]: adjusted in a follow-up PR: https://github.com/mojaloop/postman/pull/100

**Comment by eugenepolschikov** [comment]
Closing this because a follow-up PR is created: https://github.com/mojaloop/postman/pull/100

**Pull request closed without merging**